templates/federated_reporting/cfsecret.py

```diff
@@ -0,0 +1,36 @@
+#!/var/cfengine/bin/cfengine-selected-python
```

Comment:
Sure. It doesn't hurt much if we always run from policy with `cfengine-selected-python script.py` but having it here certainly makes manual debugging easier. So that it uses the "expected" version.